commit f6c72fe84c8e643a818d15ce45d60cefdd938cde (refs/changes/57/357/6)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2016-08-26T14:12:54-07:00 (3 years, 1 month ago)
    
    OS-5634 system-log SMF method has existential confusion
    Reviewed by: Alex Wilsin <alex.wilson@joyent.com>
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/overlay/generic/lib/svc/method/system-log b/overlay/generic/lib/svc/method/system-log
index f8a60a36..c2424870 100644
--- a/overlay/generic/lib/svc/method/system-log
+++ b/overlay/generic/lib/svc/method/system-log
@@ -103,20 +103,21 @@ fi
 
 if [ -f /etc/rsyslog.conf ]; then
 	if [ ! -x /usr/sbin/rsyslogd ]; then
-		echo "/etc/rsyslog.conf is missing. Exiting."
+		echo "Executable /usr/sbin/rsyslogd is not found. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 else
 	#
 	# Fall back to old syslogd
 	#
+	echo "/etc/rsyslog.conf is missing. Using syslogd instead."
 	if [ ! -f /etc/syslog.conf ]; then
 		echo "/etc/syslog.conf is missing. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 
 	if [ ! -x /usr/sbin/syslogd ]; then
-		echo "Executable /usr/sbin/syslogd not found. Exiting"
+		echo "Executable /usr/sbin/syslogd not found. Exiting."
 		exit $SMF_EXIT_ERR_CONFIG
 	fi
 fi
